### DISCUSSION TITLE
How exactly should I update Obtainium?

### DISCUSSION THREAD

**Discussion by DeltaEpsilon19498** [ImranR98/Obtainium#1984] [Q&A] [answered]
I want to get Obtainium from Github, but if I do that, I am confused on whether or not the app updates itself. Or do I need to set it up so that I would download Obtainium from Obtainium? Or do I need to use F-Droid to keep track of and download updates?

**Comment by ImranR98** [comment, marked as answer]
Obtainium is already setup to update itself (it is already included in its own apps list upon install).

**Reply by DwainZwerg** [reply]
It depends, though, on which release you choose: If you select ‘-f-droid-’, Obtainium is not entered in the app, and you can only update it via F-Droid (you can, however, manually add Obtainium as an F-Droid app). If you pick the normal release, Obtainium will update itself.

**Reply by tkgo11** [reply]
> It depends, though, on which release you choose: If you select ‘-f-droid-’, Obtainium is not entered in the app, and you can only update it via F-Droid (you can, however, manually add Obtainium as an F-Droid app). If you pick the normal release, Obtainium will update itself.

You can add it manually if you are using F-Droid version.
